docs/index.html

```diff
@@ -181,10 +181,12 @@
                   this.handleReqFailure(e);
                   return;
                 }
-                const reMajorVersion = /v(\d+)/;
+
+                const excludedTagVersions = new Set(['v0.7', 'v0.8.1']);
```

Comment:
Thanks for the update, this is indeed a much simpler diff :laughing: I don't recall off hand what the browser compatibility breakdown is for `Set`'s, although for that matter I don't know if there's any potential compatibility issues elsewhere within this page.

Do you know what the compatibility story is for Set.has? I suppose we could fallback to an array with an `includes` or `indexOf` if necessary, or even and object with key checks.

These are both really small data collections so the browser compatibility weighs more heavily than performance and/or ergonomic gains of newer constructs

Reply:
I was using the MDN compatibility chart for determining if there were any issues with utilizing the set and set.has functionality: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

It seemed pretty widely adopted so I went with it, but I'm totally open with using the array functions if we wanted to go that route. I guess like, what were we hoping to achieve with compatibility with the page? What are the thresholds we are shooting for with browser versions getting supported?

Reply:
> I guess like, what were we hoping to achieve with compatibility with the page? What are the thresholds we are shooting for with browser versions getting supported?

Great question, and to be perfectly honest it's completely undefined to the best of my knowledge. The fact that this will even work on IE (albeit 11) is more than good enough for me.